testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/AbstractHttpServerTest.java

```diff
@@ -299,6 +300,20 @@ void captureRequestParameters() {
     assertTheTraces(1, null, null, spanId, "POST", CAPTURE_PARAMETERS, response);
   }
 
+  @Test
+  void nestedPath() {
+    assumeTrue(Boolean.getBoolean("testLatestDeps"));
+    assumeTrue(options.testNestedPath);
+    String method = "GET";
+    AggregatedHttpRequest request = request(NESTED_PATH, method);
+    AggregatedHttpResponse response = client.execute(request).aggregate().join();
+    assertThat(response.status().code()).isEqualTo(NESTED_PATH.getStatus());
+    assertThat(response.contentUtf8()).isEqualTo(NESTED_PATH.getBody());
+    assertResponseHasCustomizedHeaders(response, NESTED_PATH, null);
+
+    assertTheTraces(1, null, null, null, method, NESTED_PATH, response);
+  }
```

Comment:
can this test be moved down into the webflux test class? (and then wouldn't need options.testNestedPath)

Reply:
yes, done.